### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by Yongxuanzhang**

### CODE REVIEW

File: encoder.py [hunk 1]
```diff
@@ -108,6 +108,13 @@ def _get_pooling(self, feature_map: 'np.ndarray') -> 'np.ndarray':
 
     @requests
     def encode(self, docs: Optional[DocumentArray], parameters: Optional[Dict] = None, **kwargs):
+        """
+        Encode image data into a ndarray of `D` as dimension, and fill the embedding of each Document.
+
+        :param docs: DocumentArray containing images
+        :param parameters: parameters dictionary
+        :return: an ndarray in size `B x D`.
```
Comment (JoanFM) [inline encoder.py]: this is false, does not return anything

File: encoder.py [hunk 2]
```diff
@@ -108,6 +108,13 @@ def _get_pooling(self, feature_map: 'np.ndarray') -> 'np.ndarray':
 
     @requests
     def encode(self, docs: Optional[DocumentArray], parameters: Optional[Dict] = None, **kwargs):
```
Comment (JoanFM) [inline encoder.py]: parameters is not Optional, it will always exist

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: encoder.py (hunk 2 above)
Comment (Yongxuanzhang) [reply encoder.py]: thanks!

File: encoder.py [hunk 3]
```diff
@@ -107,7 +107,13 @@ def _get_pooling(self, feature_map: 'np.ndarray') -> 'np.ndarray':
         return self.pool_fn(feature_map, axis=(2, 3))
 
     @requests
-    def encode(self, docs: Optional[DocumentArray], parameters: Optional[Dict] = None, **kwargs):
+    def encode(self, docs: DocumentArray, parameters: Dict, **kwargs):
```
Comment (JoanFM) [inline encoder.py]: docs are indeed Optional

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — APPROVED** [review]

**Pull request merged**